float64 x
float64 y
float64 z
float64 roll
float64 pitch
float64 yaw
float64 front
float64 back
float64 up
float64 left
float64 right
float64 zrange
float64 deltaX
float64 deltaY
float64 outlierCount
float64 accX
float64 accY
float64 accZ
float64 gyroX
float64 gyroY
float64 gyroZ
float64 ref_x
float64 ref_y
float64 ref_z
float64 ref_roll
float64 ref_pitch
float64 ref_yaw
